Connectors/Rhino/Speckle.Connectors.GrasshopperShared/HostApp/Helpers.cs

```diff
@@ -213,3 +214,48 @@ public static string GetParamTopology(IGH_Param param)
     return topology;
   }
 }
+
+public static class BakingHelpers
```

Comment:
we can leave this in for now, but I've introduced an `ApplyToObjectAttributes()` method on the property param in my nested props pr, we should try to refactor more once that is in to reduce duplicate object attribute code